build.gradle.kts

```diff
@@ -29,6 +29,7 @@ dependencies {
     testImplementation("org.springframework.boot:spring-boot-starter-test")
     testImplementation("org.jetbrains.kotlin:kotlin-test-junit5")
     testRuntimeOnly("org.junit.platform:junit-platform-launcher")
+    testImplementation("io.rest-assured:rest-assured:5.3.1")
```

Comment:
In our previous missions, we didn't use this library when making tests.
What do you think this library brings to us when testing our application?
I ask this because I think knowing our tools and why we need it is as important as knowing how to use it :)